app/views/schools/_prompt_join_programme.html.erb

```diff
@@ -1,11 +1,12 @@
 <% if school %>
   <% if programme_type = school.suggested_programme_types.first %>
     <%= component 'info_bar',
-        status: :neutral,
-        style: local_assigns[:style],
-        title: t("schools.prompts.join_programme.message_html", count: programme_type.activity_count, title: programme_type.title),
-        icon: fa_icon('tasks fa-3x'),
-        buttons: { t("common.labels.start") => programme_type_path(programme_type) }
-    %>
+                  status: :neutral,
+                  style: local_assigns[:style],
+                  title: t('schools.prompts.join_programme.message_html',
+                           count: programme_type.activity_types_for_school(school).length,
```

Comment:
Changed this count as it was counting completed activities rather than unique activity_types, which can lead to text such as:

You've recently completed 14 activities that are part of the Beat your baseload programme. Do you want to enroll in the programme?

When there are only 7 activity types in the programme.